src/webgl/loading.js

```diff
@@ -1129,4 +1130,183 @@ p5.prototype.model = function(model) {
   }
 };
 
-export default p5;
+/**
+ * Load a 3d model from an OBJ or STL string.
+ *
+ * <a href="#/p5/loadModel">createModel()</a> should be placed inside of <a href="#/p5/preload">preload()</a>.
```

Comment:
I think we don't need this sentence, since loading from a string means we don't have to asynchronously load the source code and everything happens immediately.